modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/authorization/enums/ResourceTypes.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package io.ballerina.messaging.broker.auth.authorization.enums;
+
+/**
+ * Enum used to represent broker resource types.
+ */
+public enum ResourceTypes {
```

Comment:
Shall we rename to `ResourceType`

Reply:
This should be done for all the enum classes in  `...authorization.enums` package

Reply:
Fixed in b30ae14